### DISCUSSION TITLE
Амнистия в честь нового года!

### DISCUSSION THREAD

**Discussion by Hardcore-punk** [octothorp-team/public-dobrograd#839] [Предложения]
Дорогая Администрация проекта, обращаюсь к вам от лица всей репрессированной шпаны Доброграда, дайте сидельцам шанс(кроме тех кто читы юзал). Те кто осознал свои ошибки, следовательно научился на них, а дураков пусть будет ждать вечная смерть без возможности выкупа и разбана. Дайте братве вашего проекта шанс, все таки такой проект уникален на территории СНГ сообщества гмода. У меня все, согласных прошу отписаться под моими словами!

**Comment by BNIDA1** [comment]
Предлагаю еще выдать всем по M249 и чтобы FK удалили как правило да

**Reply by Hardcore-punk** [reply]
ну это все твои фантазии, а действительно честным и порядочным арестантам можно и подсобить
